.travis.yml

```diff
@@ -18,4 +18,7 @@ script:
   - staticcheck ./...
   - unused ./...
   - go test -v ./...
```

Comment:
nit: this should probably be in `test.sh`, too. Note that `test.sh` correctly handles `vendor`, while this does not (and this would be obvious if this line were in that file).